### PR TITLE
Improve wording and fix tiny typos (intro & chapter 1)

### PR DESCRIPTION
**Pull request by chaozg**
I found some tiny typos along my reading so I corrected them : )
BTW, will the toolbox picture in `introduction_overview_of_cuqipy.md` have any license issue?

### CODE REVIEW

**Review by amal-ghamdi — CHANGES_REQUESTED** [review]
Hi @chaozg, many thanks for your edits. This is really helpful! I added two small comments

File: chapter01/Intro_example.ipynb [hunk 1]
```diff
@@ -215,15 +215,18 @@
     "## <font color=#CD853F>The prior </font> <a class=\"anchor\" id=\"r-the-prior\"></a>\n",
     "\n",
     "### Bayesian approach: Use prior to express belief about solution\n",
-    "- Common choice for simplicity: Gaussian prior\n",
+    "A Common choice for simplicity is the zero mean isotropic Gaussian prior, defined as\n",
```
[the remark this answers is not in the document]
Comment (chaozg) [reply chapter01/Intro_example.ipynb]: updated to `A Common choice for simplicity is the zero mean Gaussian prior where the components are independent and identically distributed (i.i.d.):`

File: chapter01/Intro_example.ipynb [hunk 2]
```diff
@@ -612,7 +615,7 @@
     "    - Define the data distribution in CUQIpy and call it `b_mult`. Plot its PDF (condition on `particular_x`). Tip: you can use lambda function to define the `Gaussian` covariance `lambda x : 0.1*(A@x)**2`.\n",
     "    - Plot the PDF of the data distribution `b_mult` given `x=particular_x`.\n",
     "    - In general, for the given `x_particular`, how does the noise to the exact data ratio change in the case of `b_mult` compared to `b`? \n",
-    "    - Plot the PDF of the data distribution `b_mult` given a different value of `x=np.array([0.4, 0.3])`. What can we say about the noise ratio to the exact data in this case? and why?\n",
+    "    - Plot the PDF of the data distribution `b_mult` given a different value of `x=np.array([0.4, 0.3])`. What can we say about the noise ratio to the exact data in this case, and why?\n",
```
[the remark this answers is not in the document]
Comment (chaozg) [reply chapter01/Intro_example.ipynb]: I see and just changed it back to what it was

**Review by amal-ghamdi — APPROVED** [review]
Thanks @chaozg! LGTM

**Pull request merged**